script/Ciel/Day 13/13_01_01_CIEL13_4.txt

```diff
@@ -86,17 +86,17 @@ With my throat layed bare and defenseless, I raise my head and take in a deep br
 [sha:e7fae0311ddbc02b95ace77f963d137eecf22bb9]{
 -- Page 316, Offset 34111.
 -- 見上げれば、壁の隙間から月明かりがうかがえた。
-When I look up, I see the moonlight through the cracks in the wall.
+When I look up, I see the moonlight peeking through the cracks in the wall.
 }
 [sha:52115845560d240e7182c1bb20941e817b9f20ee]{
 -- Page 316, Offset 34112.
 -- 体力も精神もすり潰したからか。
-Maybe it's because my strength and spirits are worn out.
+Maybe it's because both my mental and physical strength have already been completely exhausted.
 }
 [sha:73742763ac5ec3093f32c1df3d3dd9840dbda09a]{
 -- Page 316, Offset 34113.
 -- 目に見えるものすべてに靄がかかって、<虚|うつ>ろに見える。
-Everything I see seems covered in a haze and empty.
+But right now, everything in my sight looks shrouded in haze and empty.
```

Comment:
can't think of anything sufficiently poetic 
everything in sight looks empty and shrouded in a haze.